examples/application_epg/main.tf

```diff
@@ -31,4 +42,25 @@ resource "aci_application_epg" "fooapplication_epg" {
   pref_gr_memb           = "exclude"
   prio                   = "unspecified"
   shutdown               = "no"
-}
\ No newline at end of file
+}
+
```

Comment:
I suggest we add a comment here for a clear distinction from the above code snippet->

**The following depicts an example to create and associate an application EPG with the common Tenant's BD and VRF**

Reply:
Quick suggestion: we do not need 2 tenants in one main.tf 

the above file can be changed to ->

resource "aci_tenant" {}
resource "aci_vrf"
resource "aci_bridge_domain" {}
resource "aci_application_profile" {}
resource "aci_application_epg" {}

Reply:
There's only one resource which is creating a tenant. Tenant "common" is queried using the data source.